src/examples/alert/Email.cpp

```diff
@@ -60,8 +60,8 @@ int main(int , char**) {
     return 0;
 
     cout << "jwSMTP library demo program\n\n";
```

Comment:
Please remove all the jwSMTP, it was an older lib that we tested, it's not being used anymore.